### ISSUE TITLE
The installations is stuck when building a new human database

### ISSUE THREAD

**Issue by leonorrib** [FEI38750/MTD#4]
Dear authors,

I've been trying to install the MTD software for a few times but I'm always encountering the same issue which is that when creating the new human db the installations just stays trying to do so but it doesn't move on. The expected file with the human database is emtpy all the time. Do you have any idea on how to fix it or how I could figure out what's the problem behind this?

Here I share with you the latest messages provided by the installation:

```
Building a new DB, current time: 02/18/2023 17:22:40
New DB name:   /path/MTD/human_blastdb/human_blastdb
New DB title:  /path/MTD/hisat2_index_human/genome.fa
Sequence type: Nucleotide
Deleted existing Nucleotide BLAST database named /path/MTD/human_blastdb/human_blastdb
Keep MBits: T
Maximum file size: 1000000000B

```

Best regards,
Leonor

**Comment by leonorrib** [comment]
Hello :) @FEI38750 

Could you look into the issue I opened two weeks ago? I'm still having that issue :)

Have a nice day,
Leonor

**Comment by FEI38750** [comment]
Could you please try if "makeblastdb" functional by typing

> conda activate MTD
> makeblastdb

**Comment by leonorrib** [comment]
Thank you for the suggestion. Should I try it in parallel to the installation? The installation is still running, just stuck at that point I mentioned above.

**Comment by FEI38750** [comment]
Yes, you can check it in parallel. How long does the installation stuck at that step?

**Comment by leonorrib** [comment]
Ok, I'll try. It's been stuck for three weeks now :)

**Comment by FEI38750** [comment]
Maybe something is wrong with the "makeblastdb". We can bypass this step if you would not use the Magic-BLAST for alignment. Still can use Hisat2 which is the default method for reads alignment.

**Comment by leonorrib** [comment]
I just checked the commands you suggested above and I would say that makeblastdb works fine, as far as I can see. This is the output of the commands:

```
(MTD) $ makeblastdb
USAGE
  makeblastdb [-h] [-help] [-in input_file] [-input_type type]
    -dbtype molecule_type [-title database_title] [-parse_seqids]
    [-hash_index] [-mask_data mask_data_files] [-mask_id mask_algo_ids]
    [-mask_desc mask_algo_descriptions] [-gi_mask]
    [-gi_mask_name gi_based_mask_names] [-out database_name]
    [-blastdb_version version] [-max_file_sz number_of_bytes]
    [-logfile File_Name] [-taxid TaxID] [-taxid_map TaxIDMapFile] [-version]

DESCRIPTION
   Application to create BLAST databases, version 2.10.1+

Use '-help' to print detailed descriptions of command line arguments
========================================================================

Error: Argument "dbtype". Mandatory value is missing:  `String, `nucl', `prot''
Error:  (CArgException::eNoArg) Argument "dbtype". Mandatory value is missing:  `String, `nucl', `prot''

```
I'll do what you suggest to skip that step. I'll post here the outcome of it.

**Comment by leonorrib** [comment]
Dear @FEI38750 ,

I completed running MTD on my data. There are some error messages. Perhaps the main one is this one:
ERROR: The MetaPhlAn2 taxonomic profile provided was not generated with the database version v30 . Please update your version of MetaPhlAn2 to v3.0.

Do you know how it can be solved?

Best regards,
Leonor
